internal/server/v2/parser.go

```diff
@@ -27,10 +27,10 @@ func splitWithDelim(expr string, delim string) []string {
 	res := []string{}
```

Comment:
maybe trim all space/newline at the beginning of function splitWithDelim?

Reply:
Oh that's more aggressive, but I guess we can be more aggressive. Done.